src/NzbDrone.Core/MediaFiles/EpisodeImport/Specifications/FullSeasonSpecification.cs

```diff
@@ -23,8 +23,8 @@ public Decision IsSatisfiedBy(LocalEpisode localEpisode, DownloadClientItem down
 
             if (localEpisode.FileEpisodeInfo.FullSeason)
             {
-                _logger.Debug("Single episode file detected as containing all episodes in the season");
-                return Decision.Reject("Single episode file contains all episodes in seasons");
+                _logger.Debug("Single episode file detected as containing all episodes in the season. Likely due to no episode parsed from the file name.");
+                return Decision.Reject("Single episode file contains all episodes in seasons. Review File Name or Manually Import");
```

Comment:
Don't title case the sentence, `Manual Import` would be a title, but manually import isn't.